cassdegrees/templates/createprogram.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base.html" %}
+
+{% block page-title %}Create Program Template{% endblock %}
+
+{% block subtitle %}Create Program Template{% endblock %}
+
+{% block content %}
+    {% if msg != None %}
+        <p class="{% if is_error %}msg-error{% else %}msg-success{% endif %}">{{ msg }}</p>
+    {% endif %}
+
+
+    <form class="anuform" action="" method="post">
+        {% csrf_token %}
+
+        <fieldset>
+            <legend>Configure Program Details</legend>
+
+            <p>
+                <label for="code">Program Code:</label>
+                <input class="text" id="code" type="text" name="code" aria-required="true" placeholder="e.g. BARTS" required>
```

Comment:
```suggestion
                <input class="text" id="code" type="text" name="code" aria-required="true" placeholder="e.g. BARTS" required value="{{ code|default:"" }}">
```

Makes the code value autofill once the page is sent